hooks/pre_gen_project.py

```diff
@@ -0,0 +1,25 @@
+"""Python module name validation."""
+import logging
+import sys
+import re
+
+logging.basicConfig()
+logger = logging.getLogger(__name__)
+
+
+def validate_python_module_name():
+    """Check that the module name given by the user is an acceptable format."""
+    module_name = "{{ cookiecutter.module_name }}"
+    # The following regex will only match valid Python module names
+    if not re.match(r"^[a-z][_a-z0-9]+$", module_name):
```

Comment:
I still don't think you want `0-9` here

Reply:
Your call if you want to get rid of it, but see my comment above regarding numbers in python module names (in case you haven't already)